contracts/src/PriceFeedImporter.sol

```diff
@@ -84,15 +109,22 @@ contract PriceFeedImporter is EventImporter {
         _onlyMoreRecentEvents(eventInfo)
     {
         // Update the latest answer.
-        currentAnswer = int256(uint256(eventInfo.log.topics[1]));
-        roundID = uint80(uint256(eventInfo.log.topics[2]));
-        updatedAt = uint256(bytes32(eventInfo.log.data));
+        uint80 roundID = uint80(uint256(eventInfo.log.topics[2]));
+        if (roundID <= latestRoundID && latestRoundID != 0) {
+            revert("roundID should be monotonically increasing");
+        }
```

Comment:
Events should be published monotonically increasingly. This is not true apparently on startup because the blocks catch-up operation is done asynchronously to the tip following one, thanks @cam-schultz !
We may want to modify this logic because of the catch-up so that it can accept any height, but not allow overwrites.

Reply:
An issue arising from that is that there might be some missing rounds although the last one is saved. Some smart contracts relying on this information to calculate i.e. the median price may get surprised by this behavior.
If we assume that any gap can only happen at the startup and that it would be filled quickly, we could save them in a "buffer" and only respond with the `getRoundData` method if none of the previous round is missing.